src/components/rocm/roc_profiler_v2.c

```diff
@@ -0,0 +1,798 @@
+#include <rocprofiler.h>
```

Comment:
I need this to get it compiling (5.7.0 RC here):

```suggestion
#include <rocprofiler/v2/rocprofiler.h>
```

please also note the usage of the `rocprofiler/` directory. ROCm changed/make it consistent that one should only need one `-I` flag (`/opt/rocm/include`) and use the ROCm component in the `#include` directive. See [here](https://docs.amd.com/en/docs-5.4.0/release.html#wrapper-header-files)

Reply:
a full patch could look like this (not tested with V1 and V2):

```diff
diff --git i/src/components/rocm/Rules.rocm.in w/src/components/rocm/Rules.rocm.in
index 0582a0c74..433ab3b03 100644 src/components/rocm/Rules.rocm.in
--- i/src/components/rocm/Rules.rocm.in
+++ w/src/components/rocm/Rules.rocm.in
@@ -20,10 +20,6 @@ endif
 COMPOBJS += rocm.o roc_dispatch.o roc_common.o $(ROCPOBJS)

 CFLAGS += -I$(PAPI_ROCM_ROOT)/include \
-          -I$(PAPI_ROCM_ROOT)/include/hsa \
-          -I$(PAPI_ROCM_ROOT)/hsa/include/hsa \
-          -I$(PAPI_ROCM_ROOT)/include/rocprofiler \
-          -I$(PAPI_ROCM_ROOT)/rocprofiler/include \
           -g

 LDFLAGS += $(LDL)
diff --git i/src/components/rocm/roc_common.h w/src/components/rocm/roc_common.h
index 0cfd95495..743078d2a 100644 src/components/rocm/roc_common.h
--- i/src/components/rocm/roc_common.h
+++ w/src/components/rocm/roc_common.h
@@ -1,7 +1,7 @@
 #ifndef __ROC_COMMON_H__
 #define __ROC_COMMON_H__

-#include <hsa.h>
+#include <hsa/hsa.h>
 #include <dlfcn.h>
 #include <unistd.h>
 #include <stdlib.h>
diff --git i/src/components/rocm/roc_profiler.c w/src/components/rocm/roc_profiler.c
index 78a6e1f0a..0c3b3bfcc 100644 src/components/rocm/roc_profiler.c
--- i/src/components/rocm/roc_profiler.c
+++ w/src/components/rocm/roc_profiler.c
@@ -5,7 +5,7 @@
  *
  */

-#include <rocprofiler.h>
+#include <rocprofiler/rocprofiler.h>
 #include "roc_profiler.h"
 #include "roc_common.h"
 #include "htable.h"
diff --git i/src/components/rocm/roc_profiler_v2.c w/src/components/rocm/roc_profiler_v2.c
index 632e71acc..780020f45 100644 src/components/rocm/roc_profiler_v2.c
--- i/src/components/rocm/roc_profiler_v2.c
+++ w/src/components/rocm/roc_profiler_v2.c
@@ -1,4 +1,4 @@
-#include <rocprofiler.h>
+#include <rocprofiler/v2/rocprofiler.h>
 #include "roc_profiler_v2.h"
 #include "roc_common.h"
 #include "htable.h"